packages/lingua-franca/src/components/index/AboveTheFold.tsx

```diff
@@ -86,9 +86,7 @@ export const AboveTheFold = () => {
       <h2>Get Started<br />With Lingua Franca</h2>
       <Row>
         <Col className="call-to-action flex-column">
-          <div className="handbook-preview">
-            <img src={require("../../assets/index/handbook.jpg").default} width={"100%"} />
-          </div>
+          <img src={require("../../../../../img/Lingua_Franca.png").default} width="100%"  />
```

Comment:
Yes, probably. I can work on that in a separate PR. That was where I left off with the PDF/EPub stuff.

My idea was to have a recognizable string, like `%IMAGE_DIR%`, that we can substitute everywhere. We can alternatively do it with symlinks, but I'm not sure that would be appropriate because symlinks tend to confuse people, and I am not sure how well they would work on Windows.